devtools/x-core/src/lib.rs

```diff
@@ -1,15 +1,28 @@
 // Copyright (c) The Libra Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use guppy::{graph::PackageGraph, MetadataCommand};
+#[macro_use]
+extern crate rental;
```

Comment:
Has the rental crate not been updated to 2018 macros?

Reply:
Unfortunately it hasn't been, it still requires this.